### PR TITLE
MNT: add CONTRIBUTING.md as next step in the osgeo community project …

### PR DESCRIPTION
**Pull request by kmuehlbauer**
…incubation process, this is just a placeholder to start with

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
While being at the FOSS4G Conference & Codesprint, @jodygarnett and me were talking about osgeo community project incubation. We did good work in the past time, so **wradlib** is already mature enough to be part of the osgeo community. @jodygarnett already set up an accept motion on the [incubator mailing list](https://lists.osgeo.org/pipermail/incubator/2016-August/003131.html). 

To finalize we need to add this CONTRIBUTING.md as laid out in the [osgeo wiki](https://wiki.osgeo.org/wiki/OSGeo_Community_Projects). It should contain something more in-depth description of how contributions are handled. See [geogig project](https://github.com/locationtech/geogig/blob/master/CONTRIBUTING.md) or [geotools project](https://github.com/geotools/geotools/blob/master/CONTRIBUTING.md) for a small and an extensive example. @heistermann we should discuss what wradlib's CONTRIBUTING.md should look like.

Also, we need to describe how contributors can get members of the development team and be part of the decision processes regarding future wradlib.

**Comment by jodygarnett** [comment]
Thanks for the chat! The page on [OSGeo Community Projects](https://wiki.osgeo.org/wiki/OSGeo_Community_Projects#Comparison_with_Incubation) has some background.

We are only focused on the first two sections of the list, the other sections are all good ideas to work on long term (and are required for incubation). So thinking about "how contributors can get members of the development team" is great - it is not required to be a community project.

**Comment by kmuehlbauer** [comment]
PyWPS also has a nice [CONTRIBUTING.rst](https://github.com/geopython/pywps/blob/master/CONTRIBUTING.rst).

**Comment by heistermann** [comment]
I agree that the PyWPS `CONTRIBUTING.md` could serve as a great template.

**Comment by kmuehlbauer** [comment]
@heistermann I added some content to the CONTRIBUTING.md. Can you review?

I would like to incorporate this also in 0.9.0, and make the release final after that. Thoughts?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+All contributions shall comply to the project [license](LICENSE.txt).
+
+## GitHub
+
+Source code, testing, bug tracking as well as docs and wiki are handled via GitHub.
+
+## Documentation
+
+* wradlib's documentation is administered in doc/-folder, in reStructuredText format.
```
Comment (heistermann) [inline CONTRIBUTING.md]: Maybe better: 

* wradlib's documentation source is contained in the `doc` directory, and mainly written in reStructuredText.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* Jupyter Notebooks (in notebooks/-folder) are utilized to extend the documentation with examples and tutorials.
```
Comment (heistermann) [inline CONTRIBUTING.md]: maybe better: (in the `notebooks` directory)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+* wradlib uses [Sphinx](http://sphinx-doc.org/), [runipy](https://github.com/paulgb/runipy) and [nbsphinx]()https://github.com/spatialaudio/nbsphinx to compile docs
+
+## Bugs
```
Comment (heistermann) [inline CONTRIBUTING.md]: Maybe rather use "Issue tracking" as header? - as the section is about issue tracking, not only bugs.

**Comment by heistermann** [comment]
Perfect - I only have some minor (and subjective) issues. See inline comments. Merge at your dicretion.

**Pull request merged**